packages/lesswrong/components/ultraFeed/settingsComponents/UltraFeedSettingsComponents.tsx

```diff
@@ -568,31 +865,263 @@ export const MultipliersSettings: React.FC<MultipliersSettingsProps> = ({
+        </p>
+        {commentDecayBiasHours.error && (
+          <p className={classes.errorMessage}>{commentDecayBiasHours.error}</p>
+        )}
+      </div>
+
+      <div className={classes.sourceWeightItem}>
+        <div className={classes.sourceWeightContainer}>
+          <label className={classes.sourceWeightLabel}>Thread First N</label>
+          <Slider
+            className={classes.sourceWeightSlider}
+            value={threadScoreFirstNSliderValue}
+            onChange={(_, val) => threadScoreFirstN.onChange(val as number)}
+            min={1}
+            max={20}
+            step={1}
+          />
+          <input
+            type="number"
+            className={classNames(classes.sourceWeightInput, {
+              [classes.invalidInput]: !!threadScoreFirstN.error
+            })}
+            value={threadScoreFirstN.value}
+            onChange={(e) => threadScoreFirstN.onChange(e.target.value)}
+            min={1}
+            max={20}
+            step={1}
+          />
+        </div>
+        <p className={classes.sourceWeightDescription}>
+          Number of top comments to consider when calculating thread score. Default: {DEFAULT_SETTINGS.resolverSettings.commentScoring.threadScoreFirstN}
+        </p>
+        {threadScoreFirstN.error && (
+          <p className={classes.errorMessage}>{threadScoreFirstN.error}</p>
+        )}
+      </div>
+
+      <div className={classes.sourceWeightItem}>
+        <div className={classes.sourceWeightContainer}>
+          <label className={classes.sourceWeightLabel}>Thread Agg.</label>
+          <select
+            className={classNames(classes.sourceWeightInput, classes.threadAggSelect, { [classes.invalidInput]: !!threadScoreAggregation.error })}
+            value={threadScoreAggregation.value}
+            onChange={(e) => threadScoreAggregation.onChange(e.target.value)}
+          >
+            <option value="sum">Sum</option>
+            <option value="max">Max</option>
+            <option value="logSum">Log Sum</option>
+            <option value="avg">Average</option>
+          </select>
+        </div>
+        <p className={classes.sourceWeightDescription}>
+          How to aggregate comment scores into a thread score. Default: {DEFAULT_SETTINGS.resolverSettings.commentScoring.threadScoreAggregation}
+        </p>
+        {threadScoreAggregation.error && (
+          <p className={classes.errorMessage}>{threadScoreAggregation.error}</p>
+        )}
+      </div>
+    </CollapsibleSettingGroup>
+  );
+};
+
+interface ThreadInterestTuningSettingsProps {
+  formValues: ThreadInterestModelFormState;
+  errors: ZodFormattedError<ThreadInterestModelFormState, string> | null;
+  onFieldChange: (field: keyof ThreadInterestModelFormState, value: number | string) => void;
+  defaultOpen?: boolean;
+}
+
+export const ThreadInterestTuningSettings: React.FC<ThreadInterestTuningSettingsProps> = ({
```

Comment:
I assume all of the settings-related code will be deleted after launch, otherwise I think the implied maintenance burden is currently extremely far over the line of being worth it.  (It would be noticeably cheaper though still not obviously worth it if it wasn't homebrewing its own form system.)

Reply:
Not "all" but I expect the threadTuning and commentScoring to mostly go.
A hypothesis is that very good configurability will be a standout feature of our feed that some users super value